### PR TITLE
UI updates from InterConnect

### PR DESCRIPTION
**Pull request by rvennam**

### CODE REVIEW

File: src/modules/demos.js [hunk 1]
```diff
@@ -176,6 +184,7 @@ export function *watchEndDemoSession() {
     yield take(END_DEMO_SESSION);
     const demoState = yield select(demoSelector);
     try {
+      yield put(logoutSuccess());
```
Comment (l2fprod) [inline src/modules/demos.js]: should logoutSuccess() comes after the code to the backend API succeed?

File: src/routes/Dashboard/components/ForecastTile/ForecastTile.jsx [hunk 2]
```diff
@@ -39,7 +40,14 @@ const ForecastTile = ({ weather }) => {
         <TableRow key={i}>
           <TableRowColumn style={styles.column}>{forecast.dow}</TableRowColumn>
           <TableRowColumn style={styles.column}>
+            <img
```
Comment (l2fprod) [inline src/routes/Dashboard/components/ForecastTile/ForecastTile.jsx]: nice that we have icons here now. Can you add it to the ShipmentCard too for consistency?

Comment (rvennam) [inline src/routes/Dashboard/components/ForecastTile/ForecastTile.jsx]: done

File: src/modules/demos.js (hunk 1 above)
Comment (rvennam) [inline src/modules/demos.js]: done.

**Pull request merged**